packages/app/src/app/pages/Sandbox/Editor/Header/index.tsx

```diff
@@ -33,13 +35,26 @@ export const Header = () => {
     >
       <Stack align="center">
         {hasLogIn ? (
-          <Link
-            variant="muted"
-            href={dashboardUrl()}
-            css={{ lineHeight: 0 /* micro adjustment */ }}
-          >
-            <DashboardIcon />
-          </Link>
+          user && user.experiments.inPilot ? (
```

Comment:
```suggestion
          user?.experiments.inPilot ? (
```

Reply:
I did this one on purpose, so that if we remove the `experiments` check in the future we still keep the `user` check, since that can be undefined.